cluster-autoscaler/cloudprovider/clusterapi/clusterapi_utils.go

```diff
@@ -264,21 +270,38 @@ func parseIntKey(annotations map[string]string, key string) (resource.Quantity,
 }
 
 func parseCPUCapacity(annotations map[string]string) (resource.Quantity, error) {
-	return parseKey(annotations, cpuKey)
+	// Checking both sets of annotations. If the upstream annotation doesn't exist,
+	// get the value of the deprecated one. If neither exist, return nil.
+	if val, err := parseKey(annotations, cpuKey); val != zeroQuantity && err == nil {
+		return val, err
+	} else if err != nil {
+		return zeroQuantity.DeepCopy(), err
```

Comment:
Can we wrap the error with context to make it easy to find which line produced the error?

Reply:
Thanks - did some tidying up on the errors being returned